### PR TITLE
gnsi binding

### PR DESCRIPTION
**Pull request by mojiiba**
This pull adds gnsi to static binding. This requires https://github.com/openconfig/ondatra/pull/82 to be merged first.

### CODE REVIEW

**Review by guoshiuan — APPROVED** [review]

**Comment by greg-dennis** [comment]
@mojiiba, can you update this PR to use the latest Ondatra release with gnsi support and ping back when done? Thanks!

File: topologies/binding/gnsi.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Google LLC
```
Comment (greg-dennis) [inline topologies/binding/gnsi.go:1]: 2023

File: topologies/binding/gnsi.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+func (g gnsiConn) Certz() certzpb.CertzClient { return certzpb.NewCertzClient(g.conn) }
+func (g gnsiConn) Credentialz() credpb.CredentialzClient {
+	return credpb.NewCredentialzClient(g.conn)
+}
+func (g gnsiConn) AccountingPull()  accpb.AccountingPullClient { return accpb.NewAccountingPullClient(g.conn) }
+func (g gnsiConn) AccountingPush()  accpb.AccountingPushClient {
+	return accpb.NewAccountingPushClient(g.conn)
+}
+var _ = binding.GNOIClients(gnoiConn{})
```
Comment (greg-dennis) [inline topologies/binding/gnsi.go:47]: gnoi->gnsi

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: topologies/binding/gnsi.go (hunk 2 above)
Comment (mojiiba) [reply topologies/binding/gnsi.go]: done

File: topologies/binding/gnsi.go (hunk 1 above)
Comment (mojiiba) [reply topologies/binding/gnsi.go]: done

**Review by greg-dennis — APPROVED** [review]

**Pull request merged**